docs/motivation.rst

```diff
@@ -49,18 +49,18 @@ With QFace these companies can be certain that QFace does not lock them into the
 Remote Services
 ===============
 
-Some projects use network communication to communicate from the HMI to the services, which might run on a different process or event networked device. QFace was not designed for remote services, as it does not define any storage types (e.g. int32, int16, int64) it knows only an int and does not define how large the int shall be. For this QFace need to rely on the author of the generators to have a defined protocol to exchange data.
+Some projects use network communication to communicate from the HMI to the services, which might run on a different process or event networked device. QFace was not designed for remote services as it does not define any storage types (e.g. int32, int16, int64), it  only knows an int and does not define how large the int shall be. For this QFace needs to rely on the author of the generators to have a defined protocol to exchange data.
 
 Complex APIs
 ============
 
-QFace is by design limited by its features. QFace shall be easy to use and the syntax easy to remember, so that also none experts can write their interface files.
+QFace is purposely designed to have limited features. The goal is to make QFace easy to use with an easy to remember syntax so that interface files can be written by none experts.
```

Comment:
Yes, I've reworded it and submitted the changes. Bear with me as I learn GitHub.